samples/highcharts/demo/horizontal-organizational-chart/demo.js

```diff
@@ -0,0 +1,192 @@
+            color: '#DEDDCF',
+            dataLabels: {
+                color: 'black'
+            }
+        }, {
+            level: 1,
+            color: '#DEDDCF',
+            dataLabels: {
+                color: 'black'
+            },
+            height: 25
+        }, {
+            level: 2,
+            color: '#DEDDCF',
+            dataLabels: {
+                color: 'black'
+            }
+        }, {
+            level: 3,
+            dataLabels: {
+                color: 'black'
+            }
+        }],
+        nodes: [{
+            id: 'Carnivora',
+            title: null,
+            name: 'Carnivora',
+            custom: {
+                info: 'Carnivora is a diverse scrotiferan order that includes over 280 species of placental mammals.'
+            }
+        }, {
+            id: 'Felidae',
+            title: null,
+            name: 'Felidae',
+            color: '#fcc657',
+            custom: {
+                info: 'Felidae is a family of mammals in the order Carnivora, <br>colloquially referred to as cats, and constitute a clade.'
+            }
+        }, {
+            id: 'Panthera',
+            title: null,
+            name: 'Panthera',
+            color: '#fcc657',
+            custom: {
+                info: 'Panthera'
+            }
+        }, {
+            id: 'Panthera pardus',
+            title: null,
+            name: 'Panthera pardus',
+            color: '#fcc657',
+            image: '../samples/graphics/horizontal-organizational-chart/panthera.png',
+            custom: {
+                info: 'Panthera is a genus within the Felidae family <br>that was named and described by Lorenz Oken in 1816 <br>who placed all the spotted cats in this group.'
+            }
+        }, {
+            id: 'Mustelidae',
+            title: null,
+            name: 'Mustelidae',
+            color: '#C4B1A0',
+            custom: {
+                info: 'The Mustelidae are a family of carnivorous mammals,<br> including weasels, badgers, otters, ferrets, martens, mink, and wolverines, among others.'
+            }
+        }, {
+            id: 'Taxidea',
+            title: null,
+            name: 'Taxidea',
+            color: '#C4B1A0',
+            custom: {
+                info: 'Taxidea'
+            }
+        }, {
+            id: 'Lutra',
+            color: '#C4B1A0',
+            custom: {
+                info: 'Lutra'
+            }
+        }, {
+            id: 'Taxidea taxus',
+            name: 'Taxidea taxus',
+            color: '#C4B1A0',
+            image: '../samples/graphics/horizontal-organizational-chart/taxidea-taxus.png',
+            custom: {
+                info: 'Taxidea taxus is a North American badger, <br>somewhat similar in appearance to the European badger, <br>although not closely related. <br>It is found in the western and central United States, <br>northern Mexico, and south-central Canada to certain areas of southwestern British Columbia.'
```

Comment:
The `<br>` breaks are unfortunate, because they don't match the width of the tooltip. For this short text, I think it is better to remove the breaks altogether and let the paragraph flow naturally. That applies to all the info texts.

<img width="345" alt="Skjermbilde 2024-02-09 kl  07 28 17" src="https://github.com/highcharts/highcharts/assets/227836/ec73bbe0-5e44-407d-a5cb-bab33fee10e9">